src/Cli/dotnet/NugetPackageDownloader/LocalizableStrings.resx

```diff
@@ -138,4 +138,10 @@
   <data name="FailedToValidatePackageSigning" xml:space="preserve">
     <value>Failed to validate package signing.</value>
   </data>
-</root>
+  <data name="FailedToFindSourceUnderPackageSourceMapping" xml:space="preserve">
+    <value>Package Source Mapping is enabled, but no source found under the specified package ID: {0}. See the documentation for Package Source Mapping at https://aka.ms/nuget-package-source-mapping for more details.</value>
+  </data>
+  <data name="FailedToMapSourceUnderPackageSourceMapping" xml:space="preserve">
+    <value>Package Source Mapping is enabled, but no source mapped under the specified package ID: {0}. See the documentation for Package Source Mapping at https://aka.ms/nuget-package-source-mapping for more details.</value>
+  </data>
```

Comment:
Suggested text edits.  For the second one you'll need to pass in an additional string format parameter, :

```suggestion
  <data name="FailedToFindSourceUnderPackageSourceMapping" xml:space="preserve">
    <value>Package Source Mapping is enabled, but no sources were configured for the specified package ID: {0}. See the documentation for Package Source Mapping at https://aka.ms/nuget-package-source-mapping for more details.</value>
  </data>
  <data name="FailedToMapSourceUnderPackageSourceMapping" xml:space="preserve">
    <value>Package Source Mapping is enabled, but the sources ({1}) mapped for the specified package ID {0} were not configured. See the documentation for Package Source Mapping at https://aka.ms/nuget-package-source-mapping for more details.</value>
  </data>
```


@baronfel @nkolev92 @zivkan Any other suggestions on the error message wording?

Reply:
This is probably a bit complex of an ask - do we know the path to the NuGet.config file that defines the mapping? My guess is somewhere between 'yes, but...' and  'no' because the config files are hierarchical. Could we instead point to the _closest_ config file with package source mapping configuration items here? We tell the users that we couldn't find any mappings for a specific package (great!) and where to learn more about it (awesome!) but not where to make the changes (womp womp :cry:).

The rest of the message is wonderful - good set-up with describing the problem and guiding the user to where information about a solution is.

Reply:
Thanks! This is a really great idea! It is definitely worth to follow up on this proposal later as it really give specific guidance for customers.😀 Currently, I'd love to update the rest of error message to include the source name which are not configured.